recipe_scrapers/_abstract.py

```diff
@@ -137,7 +137,7 @@ def ingredients(self):
     def ingredient_groups(self) -> List[IngredientGroup]:
         return [IngredientGroup(purpose=None, ingredients=self.ingredients())]
 
-    def instructions(self) -> str:
+    def instructions(self) -> Union[str, None]:
```

Comment:
Sometimes I confuse myself.  In retrospect, I don't think that a `None` value for `instructions` makes any sense at all.

A recipe _must_ have at least one instruction, almost by definition.  I'm going to adjust this in the main branch.  The [`test_multiple_instructions` test case](https://github.com/hhursev/recipe-scrapers/blob/2d7f6045a0d831912ddccc79a415349167db672c/tests/__init__.py#L64-L75) that runs during every scraper test does sorta affirm this for most cases (but it doesn't do a good job in cases where recipes contain _exactly one_ step).